app/views/catagories/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<header class="container d-flex justify-content-between  align-items-center my-3">
+    <div>
+      <h5 class="text-brand-secondary">CATEGORIES</h5>
```

Comment:
For each category, I am not able to see the icon, also when I try to click (or tap) on a category item, the application is not navigating to the transactions page for that category